src/status_im/ui/screens/browser/navigation.cljs

```diff
@@ -1,6 +1,8 @@
 (ns status-im.ui.screens.browser.navigation
-  (:require [status-im.ui.screens.navigation :as navigation]))
+  (:require [status-im.ui.screens.navigation :as navigation]
+            [status-im.models.browser-history :as browser-history]))
 
 (defmethod navigation/preload-data! :browser
   [db [_ _ {:keys [browser/browser-id]}]]
-  (assoc db :browser/options {:browser-id browser-id}))
+  (let [dont-store (browser-history/dont-store-history-on-nav-change-if-history-exists db browser-id)]
```

Comment:
there is no reason to use this method, its difficult to read, better to just check here and assoc or not

Reply:
The purpose is to keep the logic inside the model, which means the model test can check it. This was asked for in an earlier review. If the map is checked directly, the test would no longer cover a potential mistake in `navigation.cljs`.